src/Parser/DemonHunter/Havoc/Modules/Items/DelusionsOfGrandeur.js

```diff
@@ -40,8 +42,25 @@ class DelusionsOfGrandeur extends Analyzer {
 		return this.metaCooldown * this.cooldownReductionRatio || 0;
 	}
```

Comment:
This should probably be || 1 instead so it doesn't return a 0 minute cooldown with shoulders, just in case.